soda/src/main/java/it/okkam/opendata/models/soda/SodaClassification.java

```diff
@@ -50,4 +51,12 @@ public void setDomain_metadata(Map<String, String>[] domain_metadata) {
     this.domain_metadata = domain_metadata;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') it.okkam.opendata.models.soda.SodaClassification.setDomain_metadata(Map[]) may expose internal representation by storing an externally mutable object into SodaClassification.domain_metadata [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=findbugs%3AEI_EXPOSE_REP2)